lppublish/lpad.go

```
@@ -141,8 +158,7 @@ type entityResult struct {
 func (cl *charmLoader) processTips(tips []lpad.BranchTip, results chan<- entityResult) {
```

processTip would need to return an error and some kind of sentinel to trigger the break if limit is reached or continue on UniqueName failing or Revision being blank, this adds enough complexity that I prefer the solution already here.